src/client/ProposalClient.ts

```diff
@@ -260,14 +260,21 @@ class ProposalClient implements Proposal {
   }
 
   canExecute(): boolean {
-    if (this.zDAO.isRelativeMajority) return false;
+    if (!this.zDAO.isRelativeMajority) return false;
 
     const totalScore = this.scores.reduce((prev, current) => prev + current, 0);
     const totalScoreAsBN = getDecimalAmount(
```

Comment:
Have you tried using the default utils from Ethers for handling big numbers? You might be able to use something like `ethers.utils.formatUnits(BigNumber.from(totalScore, this.zDAO.votingToken.decimals)` instead of having to write your own utils

Reply:
Yes, that's available to use `formatUtils` and `parseUnits`, confirmed. Thanks